data-sources/flutter-web/config.yml

```diff
@@ -0,0 +1,37 @@
+id: flutter-web
+displayName: Flutter Web
+description: |
```

Comment:
Data source descriptions don't support markdown and are expected to be a short description. Please come up with a short/concise description for this data source

Reply:
Hi @brammerl ,
I updated the description. Could you review it and if need any changes let me know.

Thanks...